tasks/broad/ImputationTasks.wdl

```diff
@@ -80,10 +80,10 @@ task CountVariantsInChunks {
     File panel_vcf
     File panel_vcf_index
 
-    Int disk_size_gb = ceil(2*size([vcf, vcf_index, panel_vcf, panel_vcf_index], "GiB"))
     String gatk_docker = "us.gcr.io/broad-gatk/gatk:4.1.9.0"
     Int cpu = 1
     Int memory_mb = 4000
+    Int disk_size_gb = 2 * ceil(size([vcf, vcf_index, panel_vcf, panel_vcf_index], "GiB")) + 20
```

Comment:
```suggestion
    Int disk = 2 * ceil(size([vcf, vcf_index, panel_vcf, panel_vcf_index], "GiB")) + 20
```
wondering if it's worth updating this to match the variable naming we've worked towards with standardizing our runtime blocks?

Reply:
and then updating line 98 to `disks: "local-disk ${disk} HDD"`

Reply:
Hmmm, I see your point, but the naming is consistent throughout all the tasks in this file, so I am hesitant to change just this one. 
Also do we want to standardize onf including or excluding  `_gb`?

Reply:
Ah, I see what you mean. You're totally right, good to be consistent in the file. I believe we decided on excluding `_gb`. We captured the criteria from that mobbing discussion here: https://github.com/broadinstitute/warp/pull/464#issue-1025407499, I should probably track down a good README or documentation to add these guidelines to.

Reply:
No worries, thanks for checking!!